doc/topics/jinja/index.rst

```diff
@@ -1148,6 +1148,40 @@ Returns:
     }'
   }
 
+
+.. jinja_ref:: traverse
+
+``traverse``
+--------------
+
+.. versionadded:: 2018.3.3
```

Comment:
`.. versionadded:: Fluorine`

Reply:
Ok

Reply:
In fact, would it be possible to include this patch in the next minor version of salt?

As I told in the PR description, the code is trivial as it is just a matter of exposing the existing traverse_dict_and_list function as a jinja filter through the @jinja_filter('traverse') decorator.
So it seems safe to me to merge it in the 2018.3 branch instead of the develop branch.